libretroshare/src/util/rsdir.cc

```diff
@@ -327,14 +327,14 @@ bool RsDirUtil::copyFile(const std::string& source,const std::string& dest)
 #else
 	FILE *in = fopen64(source.c_str(),"rb") ;
 
-	if(in == NULL)
+	if(in == nullptr)
```

Comment:
This and similar should be replaced by `if(!in)`

Reply:
Done. Will be more concern now. ;)